### ISSUE TITLE
Cannot parse privateKey

### ISSUE THREAD

**Issue by stevenkoelma** [Natizyskunk/vscode-sftp#250]
With the config below, I can't download my project in a wsl ubuntu environment. I get this error: 

`Cannot parse privateKey: Unsupported OpenSSH private key type: ssh-ed25519.`

I'm sure that my key is added to the server. Within wsl I can login with ssh. Also on windows with filezilla, with the same key.

Is this right? Can I do something else? Or do you need more information?

```json
**settings**
{
  "name": "name,
  "host": "provider.nl",
  "protocol": "sftp",
  "port": 2200,
  "username": "username",
  "remotePath": "/home/username/domains/wbebsite/public_html/",
  "uploadOnSave": true,
  "downloadOnOpen": true,
  "useTempFile": true,
  "openSsh": true,
  "privateKeyPath": "/home/steven/.ssh/id_ed25519",
  "passphrase": true,
  "syncOption": {
    "delete": true
  },
}
```

**Comment by flipkickmedia** [comment]
I can confirm this issue also.

In the extension log you get the following error:
`[11-02 20:54:37] [error] Error: Cannot parse privateKey: Unsupported OpenSSH private key type: ssh-ed25519`

**Comment by flipkickmedia** [comment]
Are you open to PRs?

**Comment by Natizyskunk** [comment]
@flipkickmedia,

Yes, you just have to respect [this template](https://github.com/Natizyskunk/vscode-sftp/blob/develop/.github/ISSUE_TEMPLATE/pull_request.md) 🙂
